examples/inverse/plot_mixed_norm_inverse.py

```diff
@@ -52,19 +65,19 @@
 stc_dspm = apply_inverse(evoked, inverse_operator, lambda2=1. / 9.,
                          method='dSPM')
 
-# Compute MxNE inverse solution
+# Compute (ir)MxNE inverse solution
 stc, residual = mixed_norm(evoked, forward, cov, alpha, loose=loose,
                            depth=depth, maxit=3000, tol=1e-4,
                            active_set_size=10, debias=True, weights=stc_dspm,
-                           weights_min=8., return_residual=True)
-
+                           weights_min=8., n_mxne_iter=n_mxne_iter,
+                           return_residual=True)
```

Comment:
Probably compute both MxNE and irMxNE and plot the results in a common plot_sparse_source_estimates?